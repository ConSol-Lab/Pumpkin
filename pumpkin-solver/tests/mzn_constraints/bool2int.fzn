var bool: l :: output_var;
var 0..3: x1 :: output_var;

constraint bool2int(l, x1);

solve satisfy;
