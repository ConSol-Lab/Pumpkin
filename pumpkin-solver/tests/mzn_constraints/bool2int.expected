l = false;
x1 = 0;
----------
l = true;
x1 = 1;
----------
==========
